src/pages/Contact/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Helmet } from 'react-helmet-async';
+import Link from 'components/generic/Link';
+import Twitter from 'components/Twitter';
+
+import cssBinder from 'styles/cssBinder';
+
+import ipro from 'styles/interpro-vf.css';
+import fonts from 'EBI-Icon-fonts/fonts.css';
+
+const css = cssBinder(fonts, ipro);
+
+const Contact = () => {
+  return (
+    <section className={css('vf-stack', 'vf-stack--400')}>
+      <Helmet>
+        <title>Contact us</title>
+      </Helmet>
+
+      <h3>Contact us</h3>
+      <p>Please use one of the following channels to get in touch with us.</p>
+
+      <h4>InterPro</h4>
+      <ul className={css('vf-list')}>
+        <li>
+          <Link
+            href="//www.ebi.ac.uk/support/interpro"
+            target="_blank"
+            withReferrer
+          >
+            <span className={css('icon', 'icon-common')} data-icon="&#x6e;" />{' '}
+            Submit a ticket
+          </Link>
+        </li>
+        <li>
+          <Link href="mailto: interhelp@ebi.ac.uk">
```

Comment:
I wouldn't add the helpdesk email address, as this is likely to lead to receive more spams from robots